Libraries/Components/Switch/Switch.d.ts

```diff
@@ -37,10 +38,13 @@ export interface SwitchPropsIOS extends ViewProps {
   tintColor?: ColorValue | undefined;
 }
 
-export interface SwitchChangeEvent extends React.SyntheticEvent {
+export interface SwitchChangeEventData extends TargetedEvent {
```

Comment:
Why do we need to extend from `TargetedEvent`, doesn't NativeSyntheticEvent already provide `target: number`?

Reply:
> NativeSyntheticEvent already provide target: number?

yes it's provide `event.target` only!

but  `extends TargetedEvent` needs to provide it on level of `nativeEvent`:
 
`event.nativeEvent.target // number`

Reply:
I see! I see that our Flow types also don't mention a `target` on the `nativeEvent`. Are our Flow types also incorrect? 

See here the Flow types: https://github.com/facebook/react-native/blob/5c5220a46d05fa070e9f5f990bd741efa2b09309/Libraries/Components/Switch/Switch.js#L27

Reply:
yes, they are

Reply:
> Are our Flow types also incorrect?

yes, they are

I added `target: number,`